static/js/client.js

```diff
@@ -50,6 +50,7 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
     , delta = Nightscout.plugins('delta')
     , direction = Nightscout.plugins('direction')
     , errorcodes = Nightscout.plugins('errorcodes')
+    , translate = Nightscout.language.translate
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['translate' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/797646481/issues/source?bid=2452379&fileBranchId=2723722#l53)